app/models.py

```diff
@@ -154,13 +153,6 @@ class Request(models.Model):
     mentor = models.ForeignKey(User, on_delete=models.CASCADE, null=True,
                                blank=True, related_name='mentor')
 
-    STATUSES = [
```

Comment:
I don't have much context on why this was deleted.

Reply:
it was a duplicate for some reason, asked Delores to send me her first branch so that I could check the migrations, then noticed this part too. Mistakenly removed the str representation, so I added it back

Reply:
alright